sci-libs/lib2bit/lib2bit-0.2.1.ebuild

```diff
@@ -17,15 +17,20 @@ IUSE="static"
 DEPEND=""
 RDEPEND="${DEPEND}"
 
+PATCHES=(
+	"${FILESDIR}"/${P}_respect_LDFLAGS.patch
+	"${FILESDIR}"/${P}_test_support_python3.patch
+)
+
 src_prepare(){
 	default
-	epatch "${FILESDIR}"/${P}_respect_DESTDIR.patch
-	sed -e 's#/usr/local#/usr#' -i Makefile || die
 }
 
 src_install(){
-	emake DESTDIR="${ED}" install
-	if not use static; then
-		rm "${ED}"/usr/lib/lib2bit.a || die
+	emake
```

Comment:
I doubt it work for Prefix.

Reply:
`dolib` and `doheader` are used. If they work for Prefix, this would as well.